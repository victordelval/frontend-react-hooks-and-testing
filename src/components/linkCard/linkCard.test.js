import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import LinkCard from "./linkCard";

configure({ adapter: new Adapter() });

describe("<LinkCard />", () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<LinkCard to="/"  />);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
