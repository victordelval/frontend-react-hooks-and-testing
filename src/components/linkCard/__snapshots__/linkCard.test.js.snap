// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinkCard /> renders correctly 1`] = `ShallowWrapper {}`;
